README.md

```diff
@@ -31,8 +31,8 @@ To further configure your plugin, use the following custom parameters in your `s
 | `addLayers`          | Whether to install the Datadog Lambda library as a layer. Defaults to `true`. Set to `false` when you plan to package the Datadog Lambda library to your function's deployment package on your own so that you can install a specific version of the Datadog Lambda library ([Python][4] or [Node.js][5]). |
 | `logLevel`           | The log level, set to `DEBUG` for extended logging. Defaults to `info`.                                                                                                                                                                                                                                                                                                                           |
 | `enableXrayTracing`  | Set `true` to enable X-Ray tracing on the Lambda functions and API Gateway integrations. Defaults to `false`.                                                                                                                                                                                                                                                                                   |
-| `enableDDTracing`    | Enable Datadog tracing on the Lambda function. Defaults to `true`. When enabled, it's required to set the `forwarder` parameter.                                                                                                                                                                                                                                                                         |
-| `forwarder`          | Setting this parameter subscribes the Lambda functions' CloudWatch log groups to the given Datadog forwarder Lambda function. Required when `enableDDTracing` is set to `true`.                                                                                                                                                                                                                 |
+| `enableDDTracing`    | Enable Datadog tracing on the Lambda function. Defaults to `true`. When enabled, it's required to set either the `forwarderArn` or `forwarder` parameter there is no difference in which you choose but please only use one .                                                                                                                                                                                                                                                                         |
```

Comment:
I would remove the bit on the end about the forwarder parameter and just mention the forwarderArn parameter.